src/Pages/ProductList/Components/Modal.scss

```
@@ -0,0 +1,75 @@
+@import '../../../Styles/common.scss';
+@import '../../../Styles//reset.scss';
+
+.modal-container {
+  @extend %flexCenter;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 3000;
+  background-color: rgba(0, 0, 0, 0.6);
+
+  .cart-box {
+    display: grid;
+    box-sizing: border-box;
+    padding: 40px 30px 30px 30px;
+    width: 450px;
+    height: 340px;
+    background-color: white;
+    border-radius: 10px;
+  }
+  .cart-product-name {
+    margin-bottom: 10px;
+  }
+
+  .cart-quantity {
+    align-items: center;
+    .quantity-button {
```

```suggestion
    align-items: center;
    
    .quantity-button {
```
새로운 셀렉터 앞에서는 한줄 띄워주세요

---

수정하였습니다!